components/accordion/_accordion-custom.scss

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Do not use custom class in components folder.
Add the custom styles inside the component_lab example.